db/migrate/20240206085104_remove_enums.rb

```diff
@@ -0,0 +1,73 @@
+class RemoveEnums < ActiveRecord::Migration[7.1]
+  def up
+    change_column(:storages, :health_status, :string, default: 'pending')
+    add_check_constraint(:storages,
+                         'health_status IN (\'pending\', \'healthy\', \'unhealthy\')',
+                         name: 'storages_health_status_check')
+
+    change_column(:last_project_folders, :mode, :string, default: 'inactive')
+    add_check_constraint(:last_project_folders,
+                         'mode IN (\'inactive\', \'manual\', \'automatic\')',
+                         name: 'last_project_folders_mode_check')
+
+    change_column(:project_storages, :project_folder_mode, :string)
+    add_check_constraint(:project_storages,
+                         'project_folder_mode IN (\'inactive\', \'manual\', \'automatic\')',
+                         name: 'project_storages_project_folder_mode_check')
+
+    change_column(:delayed_job_statuses, :status, :string, default: 'in_queue')
+    add_check_constraint(:delayed_job_statuses,
+                         'status IS NULL OR status IN (\'in_queue\', \'error\', \'in_process\', \'success\', \'failure\', \'cancelled\')',
+                         name: 'delayed_job_statuses_status_check')
+
+    execute <<~SQL.squish
+      DROP TYPE public.delayed_job_status RESTRICT;
+      DROP TYPE public.project_folder_modes RESTRICT;
+      DROP TYPE public.storage_health_statuses RESTRICT;
+    SQL
+  end
+
+  def down
+    execute <<~SQL.squish
+      CREATE TYPE public.delayed_job_status AS ENUM (
```

Comment:
Same issue here. We should not qualify the schema so this works in the m10y context.